clone.js

```diff
@@ -10,7 +12,7 @@ readline.question(`Wat is de naam van de nieuwe webcomponent? `, (naam) => {
   readline.question(`Wat is de description van de component? `, (description) => {
     const currentPath = process.cwd();
     const defaultPath = path.resolve(currentPath, '../webcomponent-vl-ui-' + naam);
-    readline.question(`Waar mag het project opgeslagen worden? [${defaultPath}]: `, (path) => {
+    readline.question(`Waar mag het project opgeslagen worden? [${defaultPath}]: `, async (path) => {
       initializeWebcomponent({
         naam: naam,
         description: description,
```

Comment:
Is async nodig? Er wordt namelijk niet gewacht (await) op initializeWebcomponent, dus is asynchroniteit misschien niet nodig. Ik vermoed dat je best nog `await` toevoegt.